app/gui/src/main/scala/org/bitcoins/gui/dlc/DLCPaneModel.scala

```diff
@@ -339,6 +340,43 @@ class DLCPaneModel(resultArea: TextArea, oracleInfoArea: TextArea)
                               this)
   }
 
+  def cancelDLC(status: DLCStatus): Unit = {
+    status.state match {
+      case DLCState.Offered | DLCState.Accepted =>
+        val confirmed = new Alert(AlertType.Confirmation) {
+          initOwner(owner)
+          headerText = "Confirm Canceling DLC"
+          contentText = "Are you sure you want to cancel this DLC?\n" +
```

Comment:
should probably, at the very least, include the `event_id` if not more information from the `status` in this confirm alert

Reply:
added